README.md

```diff
@@ -111,10 +113,14 @@ pkgs.catppuccin-cursors.mochaMauve
 
 1. Run the following command(s) if you have just installed, if not then look
    inside the [justfile](./justfile) to understand what commands are being run.
+   All of the following support the `-h` flag to enable building of then
+   hyprcursor theme.
 
    ```bash
    just all # Build all flavor-accent variants.
+   just all -h # Build all flavor-accent variants with hyprcursor support
    just flavor mocha # To build all variants under one single flavor.
+   just flavor mocha -h # To build all variants under one single flavor with hyprcursor support
```

Comment:
`-h` feels like the `--help` option to me so I wonder how we can better implement this :thinking: 

I wasn't entirely a fan of the `just flavor` / `just accent` approach either but it seemed to get the idea across.

Reply:
I couldn't really think of a better flag for it. It's `-h` with the "h" being for hyprcursor. Maybe it's my utter lack of experience with bash scripting, but I couldn't figure out how to make `--hyprcursor` work within the current build script. I wanted to keep it as a flag, because it's a good idea to have the xcursor route there as a fall for things like SDDM or GTK apps, which won't use the Hyprcursor even on Hyprland. If someone could explain to me how to make the build script to accept `--hyprcursor` without breaking the current options, I'll gladly change it. Or if you have a better flag I could use, I'll do that.